scripts/deploy.sh

```diff
@@ -174,6 +185,42 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
 
   cd ${OA_DIR}/playbooks/
 
+  #Distribute Magnum configuration files
+  if [[ "${DEPLOY_MAGNUM}" == "yes" ]]; then
+    cat > $OA_OVERRIDES <<'EOF'
+keystone_keystone_conf_overrides:
+  resource:
+    admin_project_name: '{{ keystone_admin_tenant_name }}'
+    admin_project_domain_name: default
+heat_policy_overrides:
+  "stacks:global_index": "role:admin"
+magnum_config_overrides:
+  certificates:
+    cert_manager_type: x509keypair
+ansible_service_mgr: "upstart"
+magnum_rabbitmq_port: "{{ rabbitmq_port }}"
+magnum_rabbitmq_servers: "{{ rabbitmq_servers }}"
+magnum_rabbitmq_use_ssl: "{{ rabbitmq_use_ssl }}"
+EOF
+    cp $OA_DIR/playbooks/roles/os_magnum/extras/env.d/magnum.yml /etc/openstack_deploy/env.d/
+    cat $OA_DIR/playbooks/roles/os_magnum/extras/haproxy_magnum.yml >> $OA_DIR/playbooks/vars/configs/haproxy_config.yml
+    cat $OA_DIR/playbooks/roles/os_magnum/extras/group_vars_magnum.yml >> $OA_DIR/playbooks/inventory/group_vars/magnum_all.yml
+    cat >> $OA_DIR/inventory/group_vars/magnum_all.yml <<'EOF'
+magnum_developer_mode: true
+magnum_git_install_branch: stable/newton
+magnum_requirements_git_install_branch: stable/newton
+pip_install_options: "--isolated"
+EOF
+    cat >> $OA_DIR/defaults/repo_packages/openstack_services.yml <<'EOF'
```

Comment:
This is causing the repo build process to build the wheels to install the mitaka version of Newton, but the --isolated pip install option above is forcing the magnum install process to ignore the repo.